codec/src/main/java/io/netty/handler/codec/compression/CompressionFormat.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.compression;
+
+/**
+ * Enumeration of possible compression formats.
+ */
+public enum CompressionFormat {
+    ZLIB,
+    GZIP,
+    DEFLATE,
+    ZLIB_OR_NONE,
+    SNAPPY,
+    BZIP2,
+    LZF,
+    LZ4,
+    FASTLZ,
+    LZMA
```

Comment:
Please add Javadoc for each enum values with a href links to its web site (or wikipedia page).